client/src/components/PaymentForm.js

```diff
@@ -68,67 +46,35 @@ const PaymentForm = ({ shoppingCart, arrivalDate, chefId }) => {
       history.push("/");
       alert("Order Placed", "success");
     }
+    closeDialog();
   };
 
-  return arrivalDate && shoppingCart.length ? (
+  return (
     <>
       <Box className={classes.paymentBox}>
         <Typography variant="h5" className={classes.paymentDetails}>
           Enter your payment details
         </Typography>
-        <CardSection className={classes.paymentDetails} />
+        <CardSection />
```

Comment:
I don't think CardSection needs to be its own component.  its only 3 lines of jsx.  it might even be able to shortened to one line of jsx.  the labels in CardSection don't seem to be used.

Reply:
Yup, agreed.